dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/dispatcher/ChannelEventRunnable.java

```diff
@@ -52,50 +52,7 @@ public ChannelEventRunnable(Channel channel, ChannelHandler handler, ChannelStat
 
     @Override
     public void run() {
-        if (state == ChannelState.RECEIVED) {

```

Comment:
Please refer to this link for why this if branch is added: https://blog.csdn.net/hengyunabc/article/details/80342163